.github/workflows/ci.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - name: make
         run: |
-          sudo apt-get update && sudo apt-get install libc6-dev-i386
+          sudo apt-get update && sudo apt-get install libc6-dev-i386 libstdc++-11-dev-i386-cross gcc-multilib g++-multilib
```

Comment:
Why do we need to add this C++ stuff in jobs where we don't use `USE_FAST_FLOAT=yes`? It looks like a mistake.

I don't see that we use `USE_FAST_FLOAT=yes` anywhere in the CI job. I think we should use it in at least one of the CI test jobs. I suggest we use it in `test-ubuntu-latest`, or maybe in most of the jobs but not all.

Reply:
This is old commit. I will add a separate job.